node-ui/src/api/dataSource/NodeDataSource.ts

```diff
@@ -675,4 +680,66 @@ export class NodeDataSource implements NodeApi {
       headers,
     );
   }
+
+  async grantCapabilities(
+    contextId: string,
+    request: CapabilitiesRequest,
+  ): ApiResponse<void> {
+    try {
+      const headers: Header | null = await createAuthHeader(
+        contextId,
+        getNearEnvironment(),
+      );
+      if (!headers) {
+        return { error: { code: 401, message: t.unauthorizedErrorMessage } };
+      }
+
+      const data = {
+        capabilities: request.capabilities,
+        signer_id: request.signer_id,
+      };
+      const url = `/contexts/${contextId}/capabilities/grant`;
+      await this.client.post<void>(
```

Comment:
```suggestion
      const response: ResponseData<void> = await this.client.post<void>(
      return response
```

To follow structure of other calls. Your API can return handled server error which "catch" won't handle